sale_order_import/models/sale.py

```diff
@@ -0,0 +1,27 @@
+# © 2016-2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    def name_get(self):
+        """Add amount_untaxed in name_get of sale orders"""
+        res = super(SaleOrder, self).name_get()
+        if self._context.get("sale_order_show_amount"):
+            new_res = []
+            for (sale_id, name) in res:
+                sale = self.browse(sale_id)
+                # I didn't find a python method to easily display
+                # a float + currency symbol (before or after)
+                # depending on lang of context and currency
```

Comment:
this looks wierd, symboll is extracted in the view dynamically

Reply:
Yes, this comment is dated from 2018, but we could let it as a reminder as having a method doing that server-side could be nice (maybe it exists now?).

Reply:
I only found `amount_to_text`, but it is not doing what we want :)
```python
>>> env.ref("base.EUR").amount_to_text(100)                                                                                         
'One Hundred Euros'
```